core-bundle/src/Routing/ResponseContext/JsonLd/ContaoPageSchema.php

```diff
@@ -69,22 +69,19 @@ public function getContext(): string
 
     public function getType(): string
     {
-        return 'contao:Page';
+        return 'Page';
     }
 
     public function toArray(): array
     {
-        $this->setProperty('contao:title', $this->title);
-        $this->setProperty('contao:pageId', $this->pageId);
-        $this->setProperty('contao:noSearch', $this->noSearch);
-        $this->setProperty('contao:protected', $this->protected);
-        $this->setProperty('contao:groups', $this->groups);
-        $this->setProperty('contao:fePreview', $this->fePreview);
+        $this->setProperty('title', $this->title);
```

Comment:
Do we even still need to override `toArray()`? We could read and write with `$this->setProperty()` and `$this->getProperty()` in our setters and getters directly now?

Reply:
Done in bf0b853e98ab9146a3e709d72c53aeeaf86dd23b